auth/authpb/auth.proto

```diff
@@ -15,3 +15,10 @@ message User {
   bytes password = 2;
   int64 tombstone = 3;
 }
+
+// Role is a single entry in the bucket authRoles
+message Role {
+  int64 ID = 1;
```

Comment:
For example, users can 1. create role A and grant A to multiple users 2. delete A 3. create a new role A. If we don't use a role ID, we need to update every (possibly many) user that is granted role A during step 2. Otherwise, users that granted A during step 1 will get the new role A created during step 3 illegally.

So I decided to use current scheme of allocating ID during creation and marking with tombstone during deletion (described in the doc). How do you think?